src/NHibernate/Cfg/MappingSchema/Hbm.generated.cs

```diff
@@ -982,7 +986,7 @@ public partial class HbmProperty {
         [System.Xml.Serialization.XmlAttributeAttribute()]
         [System.ComponentModel.DefaultValueAttribute(false)]
         public bool lazy;
-
+        
```

Comment:
But it does still avoid introducing formatting discrepancies which would could other noises on subsequent regeneration.